### PR TITLE
Integration

### PR DESCRIPTION
**Pull request by dustxd**

### CODE REVIEW

File: src/libs/Analyzer.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      } = prDetails;
+      if (!(numberOfBugs && numberOfCommits && merged_at && closed_at && created_at)) return;
+      prArray.push(prDetails);
+    });
+
+    if (prArray.length === 0) return [];
+
+    // Sort by created_at in ascending order
+    const sortedPRs = prArray.sort((a, b) => new Date(a.created_at) - new Date(b.created_at));
+    const dateRepoCreated = sortedPRs[0].created_at;
```
Comment (rarcilla) [inline src/libs/Analyzer.js:31]: so we're going with the date of when repo was created as the date of the first PR's creation? is that accurate?

**Review by rarcilla — COMMENTED** [review]
looks good to me - just unsure about using the first PR's creation date as the repo creation date.
Also when testing, my best fit lines are always straight, not sure if its because of the repos i'm choosing to analyze or something happening in the backend.

File: src/libs/Analyzer.js (hunk 1 above)
Comment (andyallin) [reply src/libs/Analyzer.js:31]: I think that's gonna be as accurate as it's gonna get since Patrick wasn't able to get the Repo creation date

**Comment by dustxd** [comment]
> looks good to me - just unsure about using the first PR's creation date as the repo creation date.
> Also when testing, my best fit lines are always straight, not sure if its because of the repos i'm choosing to analyze or something happening in the backend.

@rarcilla What do you mean by it's always straight? Like it's a perfect horizontal line?

**Comment by rarcilla** [comment]
> > looks good to me - just unsure about using the first PR's creation date as the repo creation date.
> > Also when testing, my best fit lines are always straight, not sure if its because of the repos i'm choosing to analyze or something happening in the backend.
> 
> @rarcilla What do you mean by it's always straight? Like it's a perfect horizontal line?

sorry that was unclear, yup I meant perfectly horizontal

File: src/App.jsx [hunk 2]
```diff
@@ -1,12 +1,14 @@
 import React from 'react';
 import LinkComponent from './components/LinkComponent';
+// import Data from './Visualization/EmptyData';
+// import Visualization from './Visualization/Visualization';
```
Comment (rubenprograms) [inline src/App.jsx:4]: Since you are using the Visualization component inside the LinkComponent, it is safe to remove these two from here.

**Review by rubenprograms — APPROVED** [review]

**Review by rarcilla — APPROVED** [review]

File: src/App.jsx (hunk 2 above)
Comment (dustxd) [reply src/App.jsx:4]: Sure I'll remove those!

**Pull request merged**